src/indexed_crate.rs

```diff
@@ -8,15 +8,23 @@ pub struct IndexedCrate<'a> {
 
     // For an Id, give the list of item Ids under which it is publicly visible.
     pub(crate) visibility_forest: HashMap<&'a Id, Vec<&'a Id>>,
+
+    /// As of https://github.com/rust-lang/rust/pull/105182
+    /// the `rustdoc_types::Trait`s from foreign/outside crates
+    /// (in particular, traits like `Debug`, `Send` or `Eq`) are no longer present in `inner`.
+    /// Unfortunately, the current schema requires having `rustdoc_types::Trait` for each
+    /// trait implementation.
+    /// The `dummy_trait_items` tries to fix this problem by creating a hand-written `Trait`
+    /// for each of the std traits the user can use.
+    pub(crate) dummy_trait_items: HashMap<Id, Item>,
```

Comment:
I generally dislike `dummy`-based phrasing for a variety of reasons, among which is the fact that it isn't very descriptive and its ability to communicate its purpose hinges on a lot of surrounding context. In practice that means that new contributor onboarding is more difficult than it needs to be, for both the new contributor (lots of mandatory context to learn) and for the person helping them onboard (lots of context to explain).

In general, good naming is an extremely valuable skill to practice. It makes a huge difference when programming as part of a team, which is almost all of the time.

Try `manually_inlined_builtin_traits` as a name instead?

Reply:
Done in 52c72aac72b9b642f944f07ce4bab9ddd7fb9bfb.